python/tvm/relay/frontend/keras.py

```diff
@@ -1053,22 +1053,30 @@ def _convert_simple_rnn(
     in_data = inexpr[0]
     prev_op = inexpr[1]
     weightList = keras_layer.get_weights()
-    kernel_weight = etab.new_const(weightList[0].transpose([1, 0]))
+    weightList0 = weightList[0].transpose([1, 0])
+    assert len(in_data.type_annotation.shape) == 3
+    for i in range(in_data.type_annotation.shape[1].value - 1):
+        weightList0 = np.hstack((weightList0, weightList[0].transpose([1, 0])))
```

Comment:
I'm aware of this line

Reply:
Could you plz elaborate? I still cannot see any data type issue here.

Reply:
Could you check your code for bfloat16 weights? numpy.hstack has dtype arg and I guess it possibly checks it if so numpy fails when dtype is bfloat16

Reply:
Oh I see. I will check it later. Thx

Reply:
It's true that numpy.hstack does not support bfloat16. But weightList[0] and weightList0 can never be bfloat16 to my best understanding. These two vars are from weightList = keras_layer.get_weights(), and they are NumPy arrays. If numpy does not support bfloat16, I think the dtype of weightList[0] should never be bfloat16. So this worry seems unnecessary here.